server/controllers/report-group/index.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @module email-report/report-group
+ *
+ * @description
+ * small description of this module
+ *
+ * @requires db
+ * @requires node-uuid
+ * @requires BadRequest
+ * @requires NotFound
+ */
+
+
+const uuid = require('node-uuid');
+
+const db = require('../../lib/db');
+const Topic = require('../../lib/topic');
+const moment = require('moment');
+
+const BadRequest = require('../../lib/errors/BadRequest');
+const NotFound = require('../../lib/errors/NotFound');
+const Period = require('../../lib/period');
+
+//my report function are definied in the below file
+const reports_def = require('../email-report/weeklySummaryReport');
+const email_report = require('../email-report/index');
+
+
+exports.create = create;
+exports.update = update;
+exports.list = list;
+exports.delete = remove; 
+
+/**
+ * @method create
+ *
+ * @description
+ 
+ * POST /email-report API
+ */
+function create(req, res, next) {
+
+  const sql = 'INSERT INTO report_group (code, name, description) VALUES (?,?,?);';
+
+  var reportGroup = req.body.reportGroup;
+  //
+  var values = [
+    reportGroup.code,
+    reportGroup.name,
+    reportGroup.description,
+  ];
+
+
+  db.exec(sql, values)
+    .then((row) => {
+      res.status(201).json(
+        { "code": reportGroup.code }
+      );
+    })
+    .catch(next)
+    .done();
+}
+
+
+/**
+ * @method create
+ *
+ * @description
+ 
+ * POST /email-report API
+ */
+function update(req, res, next) {
+
+  const sql = 'UPDATE report_group set code=?, name=?, description=? WHERE code=?;';
```

Comment:
The MySQL library `?` syntax allows you to do `SET` with an object for key:value pairs. 

For example: 

```js
const sql = 'UPDATE report_group SET ?';

const values = { 
   code : // ...
   name : // ... 
   description : //...
};

db.exec(sql, values)
//...
```